singer-io/tap-bing-ads
# TDL-15861 added request timeout and backoff

# Description of change
- Added request timeout and backoff for the timeout exception

# Manual QA steps
 - Unit tests to verify the API calls are retried properly
 - Unit tests to verify that the functions with no backoff are not retried
 
# Risks
 - 
 
# Rollback steps
 - revert this branch
